src/components/GitHubPRList.vue

```diff
@@ -64,21 +64,44 @@ export default {
         }
     },
     methods: {
+
+        async fetchToGithub(link) {
+            return await fetch(link, {
+                'headers': {
+                    'authorization': `token ${process.env.VUE_APP_GITHUB_AUTH_TOKEN}`,
+                },
+                'method': 'GET',
+            }).then(resp => resp.json());
+        },
+
         getPRs() {
             console.log("GitHub PR List: request", new Date());
-            let that = this;
+            this.repos.forEach(async (repo) => {
+                const pullRequests = await this.fetchToGithub(`https://api.github.com/repos/${this.user}/${repo}/pulls?state=${this.state}`);
+                for (const pullRequest of pullRequests) {
+                    const pullRequestReviews = await this.fetchToGithub(pullRequest._links.self.href + '/reviews');
+                    pullRequestReviews.reverse();
 
-            this.repos.forEach(repo => {
-                that.$http.get(`https://api.github.com/repos/${that.user}/${repo}/pulls?state=${that.state}`, {
-                    headers: {
-                        Authorization: `token ${process.env.VUE_APP_GITHUB_AUTH_TOKEN}`
+                    for (const { state } of pullRequestReviews) {
```

Comment:
Warum mit einer for-Schleife? Im Prinzip interessiert einen ja nur das neuste Review. Könnte man nicht einfach das Review mit der höchsten ID (oder das mit dem höchsten Index) nehmen und da den State rausnehmen?
Also sowas in die Richtung (nicht ausführlich getestet, könnte so aber klappen):
```
const latestReview = pullRequestReviews[pullRequestReviews.length - 1];
if (!latestReview) {
    pullRequest.review_state = 'OPEN';
} else {
    pullRequest.review_state = latestReview.state;
}
```
Das statt Zeile 83 bis 99

Reply:
Es gibt noch ein paar andere States, welche ich für nicht so relevant empfunden habe. (z.b. PENDING)
Man könnte das z.b. vor deinem Codevorschlag rausfiltern.

Reply:
@MaxGeldner  Hab den Code angepasst.